apps/server/src/shared/domain/entity/dashboard.model.entity.ts

```diff
@@ -48,18 +50,26 @@ export class DashboardGridElementModel extends BaseEntityWithTimestamps {
 	dashboard: IdentifiedReference<DashboardModelEntity>;
 }
 
+export interface IDashboardModelProperties {
+	id: string;
+	user: EntityId | User;
+}
+
 @Entity({ tableName: 'dashboard' })
 export class DashboardModelEntity extends BaseEntityWithTimestamps {
-	constructor(id: string) {
+	constructor(props: IDashboardModelProperties) {
 		super();
-		this._id = ObjectId.createFromHexString(id);
-		this.id = id;
+		this._id = ObjectId.createFromHexString(props.id);
+		this.id = props.id;
+		Object.assign(this, { user: props.user });
 	}
 
 	@OneToMany('DashboardGridElementModel', 'dashboard')
 	gridElements: Collection<DashboardGridElementModel> = new Collection<DashboardGridElementModel>(this);
 
 	// userId
+	@ManyToOne('User', { fieldName: 'userId' })
```

Comment:
Multiple dashboards per user?

Reply:
two reasons: first, we may actually want to support multiple dashboards per user, for example to nest them, or to save different states on different devices.
second, I encountered strange errors when putting a onetoone relation there :P

Reply:
I like both explanations! (: :+1: